### ISSUE TITLE
A followup 

### ISSUE THREAD

**Issue by jimmywarting** [devongovett/gif-stream#7]
Your are the first (probably only) gif streaming lib i came across, ❤️ streaming 

A followup from omggif and the issue i just created
https://github.com/deanm/omggif/issues/14
https://github.com/deanm/omggif/issues/17

Would you maybe be willing to port this to using w3c web ReadableStream? 
Adding node's stream adds quite a lot when using browserify
You could write it as if it only used ReadableStream but if you feel like having a WritableStream maybe [web-streams-polyfill](https://www.npmjs.com/package/web-streams-polyfill) will look promising

**Comment by devongovett** [comment]
Do any browsers actually implement streams yet? The node stream API is pretty ubiquitous and lots of things work with it.

**Comment by jimmywarting** [comment]
Yes, chrome and opera. Think safari TP dose it and Firefox is working on it too to implement ReadableStream

**Comment by jimmywarting** [comment]
https://jsfiddle.net/jrftnp13/4/
